ecs-entrypoints/rds-s3-backups-postgres.txt.tpl

```diff
@@ -0,0 +1,22 @@
+DATE_STRING=$(date +%Y%m%d%H%M) \
+&& mkdir -p /tmp/sqlbackups/${rds_identifier} \
+&& PGPASSWORD=\"$$DB_PASSWORD\" \
+psql \
+  -U $$DB_USERNAME \
+  -h $$DB_HOST \
+  -t \
+  -c 'SELECT datname FROM pg_database WHERE NOT datistemplate' \
+| while read DB_NAME; \
+do \
+  if [[ -n \"$$DB_NAME\" && \"$$DB_NAME\" != \"rdsadmin\" ]]; \
```

Comment:
while not important for this PR. I wonder if this shoudl move to using the grep like the mysql one.

Reply:
Okie yep can do - I'll do that when we move these to scripts within the Docker image